rllib/core/learner/learner.py

```diff
@@ -305,6 +305,8 @@ def get_weights(self, module_ids: Optional[Set[str]] = None) -> Mapping[str, Any
             A dictionary that holds the weights of the modules in a numpy-friendly
             format.
         """
+        module_states = self._module.get_state(module_ids)
```

Comment:
This method is still @abc.abstractmethod after these changes, is that intentional? I think it could also be fine for child classes.

Reply:
good catch good catch 👍